### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by JeysonFlores**
Since I was working on this I added translations for my native language. It works like a charm.
image

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+"Locación de un icono para la aplicación en una ruta absoluta o el alias del "
+"nombre de un icono."
+
+#: src/Views/EditView.vala:99
+msgid "App Categories"
+msgstr "Categorías de aplicación"
+
+#: src/Views/EditView.vala:100
+msgid "Type of the app."
+msgstr "Tipo de aplicación"
```
Comment (ryonakano) [inline po/es.po:75]: I don't understand Spanish, but are you possibly missing a trailing period here?

File: po/es.po [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+#: src/Views/FilesView.vala:24
+msgid "No valid desktop files found"
+msgstr "No se encontraron archivos de escritorios validos"
+
+#: src/Views/FilesView.vala:25
+msgid ""
+"If you've never created one, go back to Welcome View and click “New File”."
+msgstr ""
+"Si nunca has creado uno, vuelve a la sección de bienvenida y selecciona "
+"\"Nuevo Archivo\""
```
Comment (ryonakano) [inline po/es.po:111]: Same here.

File: po/es.po [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+msgstr "Editar..."
+
+#: src/Views/FilesView.vala:92
+#, c-format
+msgid "Are you sure you want to delete “%s”?"
+msgstr "¿Estás seguro que quieres eliminar \"%s\"?"
+
+#: src/Views/FilesView.vala:93
+msgid "This removes the app from Applications Menu."
+msgstr "Esto remueve la aplicación del menú de aplicaciones"
```
Comment (ryonakano) [inline po/es.po:128]: Same here.

**Review by ryonakano — COMMENTED** [review]
Thank you for translating! I have a few tiny questions, but feel free to ignore them if it's intentional because I don't understand Spanish.

**Comment by JeysonFlores** [comment]
Yeah, they were typos. They should be fixed in the last commit.

**Review by ryonakano — DISMISSED** [review]
Looks good, thank you!

**Review by ryonakano — APPROVED** [review]

**Pull request merged**